rllib/utils/pre_checks.py

```diff
@@ -24,11 +28,17 @@ def check_env(env: EnvType) -> None:
             env, (BaseEnv, gym.Env, MultiAgentEnv, RemoteBaseEnv, VectorEnv)):
         raise ValueError(
             "Env must be one of the supported types: BaseEnv, gym.Env, "
-            "MultiAgentEnv, VectorEnv, RemoteVectorEnv")
+            "MultiAgentEnv, VectorEnv, RemoteBaseEnv")
 
-    if isinstance(env, gym.Env):
+    if isinstance(env, gym.Env) and not isinstance(env, MultiAgentEnv):
```

Comment:
what if a MultiAgentEnv is a gym.Env?
shouldn't we run both check_gym_environments() and check_multiagent_environments() on it?

Reply:
I think that this might make the MultiAgentEnv template constricting because it would require making its API more rigid. For example, we have special contains and sample functions for the MultiAgentEnv, that aren't supported in a` `gym.Env`

Reply:
completely understand. what I mean is that we do:

if isinstance(env, gym.Env):
    check_gym_environments(...)
if isinstance(env, MultiAgentEnv):
    check_multiagent_environments(...)

So an env may get checked by either of these, or both of these if they implement all the APIs.
does this make sense?